precompiles/outposts/stride/events_test.go

```diff
@@ -0,0 +1,200 @@
+package stride_test
+
+import (
+	"fmt"
+	"math/big"
+
+	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/crypto"
+	cmn "github.com/evmos/evmos/v15/precompiles/common"
+	"github.com/evmos/evmos/v15/precompiles/ics20"
+	"github.com/evmos/evmos/v15/precompiles/outposts/stride"
+	"github.com/evmos/evmos/v15/utils"
+)
+
+const receiver = "stride1rhe5leyt5w0mcwd9rpp93zqn99yktsxvyaqgd0"
+
+func (s *PrecompileTestSuite) TestLiquidStakeEvent() {
```

Comment:
this should only test the `EmitLiquidStakeEvent` as a unit test, not call the `LiquidStake` method. This test is more appropriate for `tx_test.go`